### DISCUSSION TITLE
Difference between SaaS and self-hosted alternative?

### DISCUSSION THREAD

**Discussion by kirtan403** [padloc/padloc#657] [Q&A] [answered]
I just tried the self-hosted version and was not sure if the self-hosted version contains all the features of the SaaS paid plan.

Can someone from the team give some insights on this? This tool really looks cool to me! Superb UX!

**Comment by BrunoBernardino** [comment, marked as answer]
@kirtan403 the SaaS uses this same code, we only have slightly modified `docker-compose.yml` and `Dockerfile-*` files to account for the custom infrastructure modifications (for better scalability) and domains/env variables, etc.

In sum, yes, the features are the same, you might just need to make sure you enable/disable them as necessary via provisioning/config.

I hope that helps!

**Comment by kirtan403** [comment]
Hey that was insightful ! So all the SaaS paid features are available on self hosted, right?

**Comment by BrunoBernardino** [comment]
> Hey that was insightful ! So all the SaaS paid features are available on self hosted, right?

Yes. You can set your own limits.

**Comment by kirtan403** [comment]
Great, thanks for providing details
